src/challenges.js

```diff
@@ -43,6 +66,41 @@ const limitFunctionCallCount = (cb, n) => {
 // Create a cuboid object that inherits from CuboidMaker. 
 // The cuboid object must contain keys for length, width, and height.
 
+// function CuboidMaker(properties) {
+//   this.length = properties.length;
+//   this.width = properties.width;
+//   this.height = properties.height;
+// }
+// CuboidMaker.prototype.volume = function() {
+//   return `${this.lenght} * ${this.width} * ${this.height}`;
```

Comment:
I un-commented this section out and it did not work until I realized you mis-spelled `this.length`.